front/app/containers/Admin/messaging/AutomatedEmails/CampaignRow.tsx

```diff
@@ -59,10 +56,32 @@ const CampaignRow = ({
     }
   };
 
-  const isEditingEnabled = useFeatureFlag({
-    name: 'customised_automated_emails',
-  });
+  const isEditingEnabled =
+    useFeatureFlag({
+      name: 'customised_automated_emails',
+    }) &&
+    (!context || campaign.attributes.enabled);
   const isEditable = (campaign.attributes.editable_regions || []).length > 0;
+  const handleEditClick = () => {
+    if (unpersistedContextCampaign) {
+      addCampaign(
+        {
+          context,
+          campaign_name: campaign.attributes.campaign_name,
+          enabled: campaign.attributes.enabled,
+        },
+        {
+          onSuccess: (addedCampaign) => {
+            clHistory.push(
+              `${pathname}/${addedCampaign.data.id}/edit` as RouteType
```

Comment:
I couldn't find a good way to get rid of "RouteType" here. Basically, editing always corresponds to adding "/:campaign_id/edit" at the end of the URL path.